project-management/src/main/java/life/qbic/projectmanagement/domain/service/MeasurementDomainService.java

```diff
@@ -37,56 +35,20 @@ public MeasurementDomainService(MeasurementRepository measurementRepository) {
     this.measurementRepository = Objects.requireNonNull(measurementRepository);
   }
 
-  public Result<NGSMeasurement, ResponseCode> addNGS(NGSMeasurement measurement,
-      List<SampleCode> sampleCodes) {
-    try {
-      measurementRepository.save(measurement, sampleCodes);
-      return Result.fromValue(measurement);
-    } catch (RuntimeException e) {
-      log.error(
-          "Saving the NGS measurement failed for id: " + measurement.measurementCode().value(), e);
-    }
-    return Result.fromError(ResponseCode.FAILED);
-  }
-
   public List<MeasurementId> addNGSAll(
       Map<NGSMeasurement, Collection<SampleIdCodeEntry>> ngsMeasurementsMapping) {
     measurementRepository.saveAllNGS(ngsMeasurementsMapping);
     return ngsMeasurementsMapping.keySet().stream().map(NGSMeasurement::measurementId)
         .toList();
   }
 
-  public Result<ProteomicsMeasurement, ResponseCode> addProteomics(
-      ProteomicsMeasurement measurement, List<SampleCode> sampleCodes) {
-    try {
-      measurementRepository.save(measurement, sampleCodes);
-      return Result.fromValue(measurement);
-    } catch (RuntimeException e) {
-      log.error(
-          "Saving the Proteomics measurement failed for id: " + measurement.measurementCode()
-              .value(), e);
-    }
-    return Result.fromError(ResponseCode.FAILED);
-  }
-
   public List<MeasurementId> addProteomicsAll(
       Map<ProteomicsMeasurement, Collection<SampleIdCodeEntry>> proteomicsMeasurementsMapping) {
     measurementRepository.saveAllProteomics(proteomicsMeasurementsMapping);
     return proteomicsMeasurementsMapping.keySet().stream().map(ProteomicsMeasurement::measurementId)
         .toList();
   }
 
-  public Result<NGSMeasurement, ResponseCode> updateNGS(NGSMeasurement measurement) {
-    try {
-      measurementRepository.updateNGS(measurement);
-      return Result.fromValue(measurement);
-    } catch (RuntimeException e) {
-      log.error("Measurement update: Failed for measurement with id " + measurement.measurementId()
-          .value(), e);
-    }
-    return Result.fromError(ResponseCode.FAILED);
-  }
-
```

Comment:
why was this removed?

Reply:
same as for add methods - replaced by method for updating multiple measurements